{
  "Faults": [
    {
      "latitude": 50.9313844210144,
      "type": "PSI",
      "reference": "213493",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "1e345ac1-e322-4b40-94d3-05a9536ed96b",
      "minLongitude": -1.2822,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2811413908714795,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-29T11:00:00.000+0000",
      "estimatedRestoration": "2023-02-03T12:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '213493'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/213493",
      "maxLongitude": -1.2776,
      "minLatitude": 50.9298,
      "name": "SO32 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO32 2UA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2822,
              50.9321
            ],
            [
              -1.2776,
              50.9321
            ],
            [
              -1.2776,
              50.9298
            ],
            [
              -1.2822,
              50.9298
            ],
            [
              -1.2822,
              50.9321
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9321,
      "etrUpdate": true
    },
    {
      "latitude": 51.745403075114524,
      "type": "LV",
      "reference": "KM6843",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-30T10:30:00.000+0000",
      "affectedCustomerCount": 10,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "c829a292-a6b6-42b4-a8be-3742941cb69d",
      "minLongitude": -1.2574,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2373609640390861,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-30T10:18:00.000+0000",
      "estimatedRestoration": "2023-01-30T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KM6843'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KM6843",
      "maxLongitude": -1.1567,
      "minLatitude": 51.7156,
      "name": "OX44 & OX1 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "OX1 3DU",
        "OX1 3DX",
        "OX1 3DY",
        "OX1 3DZ",
        "OX44 9DF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2574,
              51.7533
            ],
            [
              -1.1567,
              51.7533
            ],
            [
              -1.1567,
              51.7156
            ],
            [
              -1.2574,
              51.7156
            ],
            [
              -1.2574,
              51.7533
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-30T10:50:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7533,
      "etrUpdate": true
    },
    {
      "latitude": 51.595862580822555,
      "type": "LV",
      "reference": "KM6855",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-30T10:45:00.000+0000",
      "affectedCustomerCount": 25,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "cc4deee9-b897-4943-aab5-950492a6ae2d",
      "minLongitude": -1.7822,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7795794093830521,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-30T12:55:00.000+0000",
      "estimatedRestoration": "2023-01-30T23:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KM6855'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KM6855",
      "maxLongitude": -1.7717,
      "minLatitude": 51.5898,
      "name": "SN2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN2 5HA",
        "SN2 5HN",
        "SN2 5HR",
        "SN2 5HS",
        "SN2 5JE",
        "SN2 5LZ",
        "SN2 7AS",
        "SN25 4AD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.7822,
              51.5986
            ],
            [
              -1.7717,
              51.5986
            ],
            [
              -1.7717,
              51.5898
            ],
            [
              -1.7822,
              51.5898
            ],
            [
              -1.7822,
              51.5986
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "c9605308-b0f4-4374-ac0a-381298cd38d7"
      ],
      "maxLatitude": 51.5986,
      "etrUpdate": true
    },
    {
      "latitude": 50.918863768955624,
      "type": "LV",
      "reference": "KM7105",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-30T15:25:00.000+0000",
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ba9617d4-2a64-4ea3-9718-75afe2bddd89",
      "minLongitude": -1.46,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4589335232527636,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-30T14:24:00.000+0000",
      "estimatedRestoration": "2023-01-31T02:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KM7105'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KM7105",
      "maxLongitude": -1.4582,
      "minLatitude": 50.9182,
      "name": "SO16 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO16 9AD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.46,
              50.9197
            ],
            [
              -1.4582,
              50.9197
            ],
            [
              -1.4582,
              50.9182
            ],
            [
              -1.46,
              50.9182
            ],
            [
              -1.46,
              50.9197
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-30T15:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9197,
      "etrUpdate": true
    },
    {
      "latitude": 51.328621430138135,
      "type": "HV",
      "reference": "KM7126",
      "etaUpdate": true,
      "affectedCustomerCount": 27,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "615642de-1bbb-420c-af52-4aaa8d530b3d",
      "minLongitude": -2.1477,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.125275994670688,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-30T15:00:00.000+0000",
      "estimatedRestoration": "2023-01-31T02:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KM7126'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KM7126",
      "maxLongitude": -2.1022,
      "minLatitude": 51.3139,
      "name": "BA14 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BA14 6BR",
        "BA14 6BS",
        "BA14 6BT",
        "BA14 6BU",
        "BA14 6BX",
        "BA14 6BY",
        "BA14 6BZ",
        "BA14 6DB",
        "BA14 6JJ",
        "BA14 6JU",
        "BA14 6LE",
        "BA14 6LF",
        "BA14 6LG",
        "BA14 6LH",
        "BA14 6LQ",
        "BA14 6LR",
        "BA14 6LT",
        "BA14 6LU",
        "BA14 6LX",
        "BA14 6LY",
        "BA14 6LZ",
        "BA14 6NA",
        "BA14 6NB",
        "BA14 6ND",
        "BA14 6NJ",
        "BA14 6NL",
        "BA14 6NT",
        "BA14 6SB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1477,
              51.3504
            ],
            [
              -2.1022,
              51.3504
            ],
            [
              -2.1022,
              51.3139
            ],
            [
              -2.1477,
              51.3139
            ],
            [
              -2.1477,
              51.3504
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3504,
      "etrUpdate": true
    },
    {
      "latitude": 50.7364757290749,
      "type": "LV",
      "reference": "KM7161",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-30T16:45:00.000+0000",
      "affectedCustomerCount": 16,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "32276694-02f2-4690-85c8-cf2dc3c3d6b5",
      "minLongitude": -1.9667,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9645923629147288,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-30T15:47:00.000+0000",
      "estimatedRestoration": "2023-01-30T23:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KM7161'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KM7161",
      "maxLongitude": -1.9614,
      "minLatitude": 50.7355,
      "name": "BH15 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH15 3HY",
        "BH15 3JB",
        "BH15 3JH",
        "BH15 3JQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9667,
              50.739
            ],
            [
              -1.9614,
              50.739
            ],
            [
              -1.9614,
              50.7355
            ],
            [
              -1.9667,
              50.7355
            ],
            [
              -1.9667,
              50.739
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-30T16:50:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.739,
      "etrUpdate": true
    },
    {
      "latitude": 51.643078699821906,
      "type": "LV",
      "reference": "KM6984",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-30T13:25:00.000+0000",
      "affectedCustomerCount": 8,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "4c28cd5c-3f8f-44b3-8a8d-d107d6598b49",
      "minLongitude": -0.7342,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7308454668553122,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-30T12:29:00.000+0000",
      "estimatedRestoration": "2023-01-30T23:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KM6984'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KM6984",
      "maxLongitude": -0.7268,
      "minLatitude": 51.6411,
      "name": "HP13 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "HP13 6BN",
        "HP13 6BW",
        "HP13 7PG",
        "HP13 7PQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7342,
              51.645
            ],
            [
              -0.7268,
              51.645
            ],
            [
              -0.7268,
              51.6411
            ],
            [
              -0.7342,
              51.6411
            ],
            [
              -0.7342,
              51.645
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.645,
      "etrUpdate": true
    },
    {
      "latitude": 50.827753055498704,
      "type": "LV",
      "reference": "KM7298",
      "etaUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "d1626e60-4cb2-48af-96d7-328044345ecb",
      "minLongitude": -1.0585,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.0562948113243356,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-30T19:30:00.000+0000",
      "estimatedRestoration": "2023-01-30T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KM7298'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KM7298",
      "maxLongitude": -1.0525,
      "minLatitude": 50.8249,
      "name": "PO3 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO3 5FU",
        "PO3 5PB",
        "PO3 5RJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0585,
              50.83
            ],
            [
              -1.0525,
              50.83
            ],
            [
              -1.0525,
              50.8249
            ],
            [
              -1.0585,
              50.8249
            ],
            [
              -1.0585,
              50.83
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.83,
      "etrUpdate": true
    },
    {
      "latitude": 51.40906583605627,
      "type": "LV",
      "reference": "KM7323",
      "etaUpdate": true,
      "affectedCustomerCount": 12,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "6c9a5dbd-6011-44ea-b493-b36c7f7d59a1",
      "minLongitude": -1.3365,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3347556448375697,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-30T20:30:00.000+0000",
      "estimatedRestoration": "2023-01-31T04:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KM7323'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KM7323",
      "maxLongitude": -1.3323,
      "minLatitude": 51.4076,
      "name": "RG14 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG14 1TS",
        "RG14 1TT",
        "RG14 1UA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3365,
              51.4102
            ],
            [
              -1.3323,
              51.4102
            ],
            [
              -1.3323,
              51.4076
            ],
            [
              -1.3365,
              51.4076
            ],
            [
              -1.3365,
              51.4102
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4102,
      "etrUpdate": true
    },
    {
      "latitude": 58.444184531373004,
      "type": "LV",
      "reference": "KM6997",
      "etaUpdate": true,
      "affectedCustomerCount": 15,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "8f945483-2471-4028-a104-f5a66263a2f1",
      "minLongitude": -3.0949,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -3.0896254652776185,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-30T12:40:00.000+0000",
      "estimatedRestoration": "2023-01-30T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KM6997'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KM6997",
      "maxLongitude": -3.0756,
      "minLatitude": 58.4374,
      "name": "KW1 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "KW1 4HA",
        "KW1 4HB",
        "KW1 4HD",
        "KW1 4HE",
        "KW1 4HP",
        "KW1 4NS",
        "KW1 5HA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -3.0949,
              58.4472
            ],
            [
              -3.0756,
              58.4472
            ],
            [
              -3.0756,
              58.4374
            ],
            [
              -3.0949,
              58.4374
            ],
            [
              -3.0949,
              58.4472
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-30T13:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 58.4472,
      "etrUpdate": true
    },
    {
      "latitude": 56.394614298970545,
      "type": "LV",
      "reference": "KM7231",
      "etaUpdate": true,
      "affectedCustomerCount": 34,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "5adab787-5541-48bc-95eb-16b4afdc0b5f",
      "minLongitude": -3.4515,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -3.4481694055234042,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-30T17:16:00.000+0000",
      "estimatedRestoration": "2023-01-30T23:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KM7231'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KM7231",
      "maxLongitude": -3.4443,
      "minLatitude": 56.393,
      "name": "PH2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PH2 0GW",
        "PH2 0GX",
        "PH2 0GY",
        "PH2 0GZ",
        "PH2 0LR",
        "PH2 0LS",
        "PH2 0LT",
        "PH2 0PB",
        "PH2 0PG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -3.4515,
              56.3956
            ],
            [
              -3.4443,
              56.3956
            ],
            [
              -3.4443,
              56.393
            ],
            [
              -3.4515,
              56.393
            ],
            [
              -3.4515,
              56.3956
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-30T17:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.3956,
      "etrUpdate": true
    }
  ]
}
